dapr/actor/client/proxy.py

```diff
@@ -98,7 +98,7 @@ class ActorProxy:
     communication.
     """
 
-    _default_proxy_factory = ActorProxyFactory()
```

Comment:
I moved this to the instance creation because it was getting invoked on import(dependency load), which was triggering the healthcheck before the fake test servers would start.